### ISSUE TITLE
skip-dirs options in .golangci.yml not equivalent to --skip-dirs flags

### ISSUE THREAD

**Issue by fredbi** [golangci/golangci-lint#235]
Entering `skip-dirs`  regexp in .golangci.yml does not exclude dirs:
```yaml
skip-dirs:
  - (assets|cognito)
``` 
=> reports issues about files there

whereas running from the same directory:
```bash
$ golangci-lint run --skip-dirs='(assets|cognito)'
``` 
skips the dirs with this pattern as expected.

1. Version of golangci-lint: `golangci-lint --version` (or git commit if you don't use binary distribution)
```bash
$ git log -1
commit 0c82b641236764940235a15c3d51a4623b8045a1
Author: Denis Isaev <denis@golangci.com>
Date:   Wed Sep 5 20:49:57 2018 +0300

    return gosec default excludes: it's too annoying for users
```
2. Go environment: `go version && go env`
```
$ go version && go env
go version go1.10.3 linux/amd64
GOARCH="amd64"
GOBIN=""
GOCACHE="/home/ubuntu/.cache/go-build"
GOEXE=""
GOHOSTARCH="amd64"
GOHOSTOS="linux"
GOOS="linux"
GOPATH="/home/ubuntu/thefundschain/poc.1.0.2"
GORACE=""
GOROOT="/usr/local/go"
GOTMPDIR=""
GOTOOLDIR="/usr/local/go/pkg/tool/linux_amd64"
GCCGO="gccgo"
CC="gcc"
CXX="g++"
CGO_ENABLED="1"
CGO_CFLAGS="-g -O2"
CGO_CPPFLAGS=""
CGO_CXXFLAGS="-g -O2"
CGO_FFLAGS="-g -O2"
CGO_LDFLAGS="-g -O2"
PKG_CONFIG="pkg-config"
GOGCCFLAGS="-fPIC -m64 -pthread -fmessage-length=0 -fdebug-prefix-map=/tmp/go-build018406683=/tmp/go-build -gno-record-gcc-switches"
``` 

3. Verbose output of running: `golangci-lint run -v`
cf. description above

**Comment by jirfag** [comment]
hi!
are you sure that you've properly use `skip-dirs` in `.golangci.yml`? it should be under `run` section, like this:
```yaml
# options for analysis running
run:
  skip-dirs:
    - src/external_libs
    - autogenerated_by_my_lib
```

**Comment by ismaelvacco** [comment]
Just an FYI:

In a recent version, the `run.skip-dirs` was deprecated. The right way is `issues.exclude-dirs`.

**Comment by caibirdme** [comment]
> Just an FYI:
> 
> In a recent version, the `run.skip-dirs` was deprecated. The right way is `issues.exclude-dirs`.

Seems like there's no `issues.exclude-dirs` option: https://golangci-lint.run/usage/configuration/#issues-configuration

**Comment by ldez** [comment]
You are reading the commentation of golangci-lint v2.

- https://ldez.github.io/blog/2025/03/23/golangci-lint-v2/
- https://golangci-lint.run/product/migration-guide/
- https://golangci-lint.run/usage/false-positives/#exclude-issues-by-path

**Comment by Emptyless** [comment]
to build on the answer of @ldez, a comparable `run.skip-dirs` or `issues.exclude-dirs` would be:
```yaml
linters:
  exclusions:
    paths:
      - assets
      - cognito
 ```

**Comment by ldez** [comment]
The answer is the same, because `run.skip-files`/`issues.exclude-files` and `run.skip-dirs`/`issues.exclude-dirs` have been merged into `linters.exclusions.paths`.
